lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -505,6 +508,21 @@ class DetailsRenderer {
     if (item.selector) element.setAttribute('data-selector', item.selector);
     if (item.snippet) element.setAttribute('data-snippet', item.snippet);
 
+    if (!item.boundingRect || !this._fullPageScreenshot) {
+      return element;
+    }
+
+    const fullPageScreenshot = this._fullPageScreenshot;
```

Comment:
nit: this doesnt seem necessary

Reply:
ah i think it was bc of line length issues in an earlier revision